packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -28,6 +28,16 @@ async function initDefaultWorkflow({ workflowsService, stagesService }) {
 
 function extendReviewWorkflowContentTypes({ strapi }) {
   const extendContentType = (contentTypeUID) => {
+    const maxContentTypeNameLength =
+      63 - 1 /* _ */ - ENTITY_STAGE_ATTRIBUTE.length - '_links_inv_fk'.length;
```

Comment:
why the - 1 ?

Reply:
because we add `_` after the content type name (that's why I put `/* _ */` but it might not be sufficient to explain 😅

Reply:
ah 💡  I thought the _ refered to the Content type name itself. I missinterpreted it